### PR TITLE
Audio notifications

### PR DESCRIPTION
**Pull request by 0phelia36**
## TL;DR
Quick summary/list of your change(s)
Sounds now play when a recipe completes, an error occurs, or a a recipe has reached a step that requires user input. User input sounds repeat every 30 seconds. Checkboxes in the settings menu allow them to be disabled individually.

## What
What is affected by this PR?
- [ ] API
- [X] GUI
- [ ] Hardware Integration
- [ ] OS/Deployment
- [ ] Documentation
- [ ] Other (please describe in notes)

## Testing
How was this tested?
- [ ] Locally Virtualized
- [X] Raspberry Pi
- [ ] With Hardware
- [ ] Other (please describe in notes)

## Notes
How was your day?

### CODE REVIEW

**Review by sparklestheunicorn — APPROVED** [review]
Love this. Thinking about some intro music now as well :3
